modules/codelite/codelite.lua

```diff
@@ -21,7 +21,7 @@
 	function codelite.cfgname(cfg)
 		local cfgname = cfg.buildcfg
 		if codelite.workspace.multiplePlatforms then
-			cfgname = string.format("%s|%s", cfg.platform, cfg.buildcfg)
+			cfgname = string.format("%s-%s", cfg.platform, cfg.buildcfg)
```

Comment:
Why `-`? The comment in the linked issue indicates that `_` worked, and that would be consistent with how `gmake`/`gmake2` handles this.

Reply:
i just didn't like seeing x86_64_Debug i thought x86_84-Debug was more pleasing